index.js

```diff
@@ -21,6 +21,10 @@ if (!heimdall.hasMonitor('async-disk-cache')) {
   heimdall.registerMonitor('async-disk-cache', function AsyncDiskCacheSchema() {});
 }
 
+var homedir = require('os').homedir().split(path.sep);
+var username = homedir[homedir.length - 1];
```

Comment:
in new node, we have a nice `os.userInfo().username`... would be nice if we had that everywhere. #someday